### PR TITLE
config: Switch camlistore errorutil to go4.org/errorutil

### PR DESCRIPTION
**Pull request by dghubble**

### CODE REVIEW

File: config/vendor/go4.org/errorutil/highlight.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 // Package errorutil helps make better error messages.
-package errorutil
+package errorutil // import "go4.org/errorutil"
```
Comment (crawford) [inline config/vendor/go4.org/errorutil/highlight.go]: Why is this needed?

Comment (dghubble) [inline config/vendor/go4.org/errorutil/highlight.go]: Its not my comment, its in upstream's source. https://godoc.org/go4.org/errorutil

Comment (crawford) [inline config/vendor/go4.org/errorutil/highlight.go]: Oh, I didn't see it in the other commits so I assumed you added it.

**Comment by crawford** [comment]
LGTM
